modules/covid_vaccine/app/jobs/covid_vaccine/expanded_submission_job.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'sentry_logging'
+
+module CovidVaccine
+  class ExpandedSubmissionJob
+    include Sidekiq::Worker
+    include SentryLogging
+
```

Comment:
I think we want to make sure that we're not retrying.

Also I'm not entirely sure how the MVI errors will be handled since it didn't look like you were raising an exception.

Reply:
add raising exceptions for both MVI cases and for the no preferred facility use case.   I need to figure out what the default behavior is wrt retries as I dont think we want to retry for any of these exceptions.  For the no facility found exception the state in the DB is updated to `enrollment_out_of_band` to reflect that manual intervention is required.  For the MVI exception cases we can let the job pick up the record on next execution to try again.